### ISSUE TITLE
Theano compatibility issue

### ISSUE THREAD

**Issue by forcecore** [transcranial/keras-js#16]
I get this error message from weblas:

%cUnhandled rejection Error: Second dimension must be of same size for input Tensors (second Tensor is transposed).

Comparing the model.json in the working example and mine, I've noticed that dimension ordering was in Theano convention (3x64x64 instead of 64x64x3). I think this should be documented, at least.

**Comment by transcranial** [comment]
@forcecore Yeah, everything was written with tensorflow in mind. There are a few places where theano dimension ordering is handled, but not everywhere yet. Definitely something that will be addressed.

**Comment by ml9951** [comment]
I'm running into the same problem, @forcecore did you come up with a workaround?

**Comment by forcecore** [comment]
Not really, I just re-trained my network in TF order for Keras-js.
